_layouts/release.html.haml

```diff
@@ -67,7 +67,7 @@
           %a.artifact{:href=>a.to_javadoc_url, :target=>'_blank'} javadoc
         %a.artifact{:href=>a.to_pom_url, :target=>'_blank'} pom
 - unless page.release_notes.nil?
-  %h3
+  %h3.release-notes
```

Comment:
The classes have been used  mostly as a form of "label" for identification purpose, since there were multiple occurrences of `<h3>` . Couldn't use `id` as it was not unique and I wanted it to be more generic for each blog.
Please suggest, if you have know of any better way, I'll fix it accordingly.